docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,242 @@
+        consumer.confluent.monitoring.interceptor.sasl.mechanism=PLAIN
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For more information, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+-----------------------------------------
+Configuration Example for Schema Registry
+-----------------------------------------
+
+    The following is a common configuration example.
+
+    **Communicate with the Confluent Schema Registry over HTTPS**
+
+    #. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
+       KSQL configuration file:
+
+       .. code:: bash
+
+            ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+
+    #. If the Schema Registry's SSL certificate is not signed by a CA that is recognized by the JVM by default, then you
+       must provide a suitable truststore when starting KSQL via the command line:
+
+       .. code:: bash
+
+          $ KSQL_OPTS="-Djavax.net.ssl.trustStore=<path-to-trust-store> -Djavax.net.ssl.trustStorePassword=<store-password>" ksql-server-start <props>
+
+    The exact settings you need depend on what SASL mechanism the Confluent Schema Registry is using is using and how your SSL certificates
+    are signed. For more information, see :ref:`schemaregistry_security`.
+
+
+Using KSQL with Kafka Clusters Secured with ACLs
+------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+Kafka clusters that use ACLs to control access to resources require clients to authenticate as a particular user. Interactive
+KSQL clusters require that this authenticated KSQL user has open access to create, read, write, delete topics, and use any
+consumer group.
+
+The :ref:`Kafka Authorizer <kafka_authorization>` (``SimpleAclAuthorizer``) terminology is used to describe the ACLs.
+Each ACL is made up of these parts:
+
+Resource
+    A resource is an object that adheres to the permissions defined by the ACL. A resource is comprised of a resource type
+    and resource name:
+
+    - ``RESOURCE_TYPE``, for example ``TOPIC`` or consumer ``GROUP``.
+    - Resource name, where the name is either specific or the wildcard ``*``, meaning all resources of this type.
+
+    The ``CLUSTER`` resource type does not require a resource name because it refers to the entire Kafka cluster.
+
+Operation
+    The operation that is performed on the resource, for example ``READ``.
+
+Permission
+    The type of access an ACL allows or denies access.
+
+Principal
+    An authenticated user or group. For example, ``"user: Fred"`` or ``"group: fraud"``. The identity of Kafka clients
```

Comment:
Not 'authenticated user', more 'specific user'. 

> The identity of Kafka clients is the user principal which represents an authenticated user in a secure cluster.

Not sure what this means.

Reply:
`specific authenticated user `?